cpu/k60/Makefile

```diff
@@ -0,0 +1,7 @@
+# define the module that is build
+MODULE = cpu
+
+# add a list of subdirectories, that should also be build
+DIRS = periph $(CORTEX_M4_COMMON) devio $(KINETIS_COMMON)
```

Comment:
Actually, it does not work when I remove them (but keep the `USEMODULE`s in Makefile.include). Also, I found that all other CPU Makefiles are specifying `DIRS = periph xxx_common etc..`. Is there something missing from my Makefile to make it work without setting DIRS?